helm-charts/konk/templates/deployment.yaml

```diff
@@ -124,6 +130,7 @@ spec:
           - mountPath: /etc/kubernetes/pki/apiserver/etcd
             name: etcd-client-certs
             readOnly: true
+        {{- end }}
```

Comment:
do tests exercise this version of installing etcd as a sidecar in the pod?

Reply:
No, rather than increasing the test matrix I just rewrote the test to use the etcd operator. We could fairly easily add this test back by supplying two values files as input to the test matrix. Naturally, that doubles the number of tests we run.